i18n/locales/it/account.json

```diff
@@ -0,0 +1,334 @@
+{
+  "action": {
+    "receive": "RICEVI",
+    "send": "INVIA"
+  },
+  "add-asset": {
+    "action": {
+      "remove": "Rimuovi",
```

Comment:
Please add a translation for `"add-asset": "Add asset to account",` below line 7

Reply:
thanks for the catch @ebma! I've just applied all the changes above in my latest commit. Sorry for changing values into curly braces {{..}} I didn't notice that

Reply:
Thanks! @andywer it might be a good idea if we added some guidelines on how to contribute new translations to the README and point out important things like 'Don't change values in curly braces, ...'?

Reply:
@ebma Good idea! Can you prepare something? We should also state what files to touch and how to test it.

Reply:
Yes I'll do it.